src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -503,6 +556,21 @@ export default {
 			}
 		},
 
+		toggleHandRaised() {
+			const raisedHand = !this.model.attributes.raisedHand
+			if (this.raisingHandNotification) {
+				this.raisingHandNotification.hideToast()
+				this.raisingHandNotification = null
+			}
+			if (raisedHand) {
+				this.raisingHandNotification = showMessage(t('spreed', 'You are now raising your hand.'))
+			} else {
+				this.raisingHandNotification = showMessage(t('spreed', 'You are no longer raising your hand.'))
+			}
```

Comment:
I'm not sure I would show a message for this at all.

Reply:
this is mostly visual feedback for an icon, for example if you'd use the keyboard shortcut.
on the other hand, for this action specifically you'd see the hand disappear at the bottom

I'll remove it